### PR TITLE
feat: add logic for ordering output

### PR DESCRIPTION
**Pull request by rjdabbar**

### CODE REVIEW

**Comment by rjdabbar** [comment]
@lorentzforces  dunno if this is logically correct, ive not done anything in his language before since like hs.

File: src/fyi/lorentz/tempBot/service/TemperatureProcessor.java [hunk 1]
```diff
@@ -48,18 +49,14 @@ public TemperatureProcessor(String text) {
     }
 
     private void
-    formatTemperature(StringBuilder buffer, Temperature temp) {
+    formatTemperature(StringBuilder buffer, Temperature temp, String firstTemp) {
         DecimalFormat format = new DecimalFormat("###,###,###,###.###");
```
Comment (lorentzforces) [inline src/fyi/lorentz/tempBot/service/TemperatureProcessor.java:53]: This needs to be declared in the scope it's used - should be moved to the handleStringFormatting method.

File: src/fyi/lorentz/tempBot/service/TemperatureProcessor.java [hunk 2]
```diff
@@ -22,11 +22,12 @@ public TemperatureProcessor(String text) {
     processMessage() {
         String[] temperatureStrings = getTemperatureStrings();
         Temperature[] temperatures = convertTemperatures(temperatureStrings);
+        String firstTemp = getTemperatureOrder(text);
```
Comment (lorentzforces) [inline src/fyi/lorentz/tempBot/service/TemperatureProcessor.java:25]: Here we're checking for the presence of the unit character in the entire message text, which is likely to grab something else that happens to have an F or C in it. This should probably be called on a per-temperature basis. 

Debating to myself whether it's better to track indexes here in a separate array, or to throw a "convertedFrom" field on Temperature that denotes what the original unit was.

**Review by lorentzforces — CHANGES_REQUESTED** [review]

**Comment by lorentzforces** [comment]
This issue has been resolved by PR #4.
Closing this PR.

**Pull request closed without merging**